composer.json

```diff
@@ -17,22 +17,24 @@
   "require": {
     "php": "^7.2",
     "ext-intl": "*",
-    "ext-json": "*"
+    "ext-json": "*",
+    "inpsyde/modularity": "^1.3.0"
   },
   "require-dev": {
     "phpunit/phpunit": "^7.1",
     "brain/monkey": "^2.3",
     "ptrofimov/xpmock": "^1",
-    "johnpbloch/wordpress-core": "^3.8 || ^4.0 || ^5.0",
+    "johnpbloch/wordpress-core": " ^5.0",
     "woocommerce/woocommerce": "^2.2 || ^3.0 || ^4.0",
     "fzaninotto/faker": "^1.9@dev",
-    "mollie/mollie-api-php": "^2.21"
+    "mollie/mollie-api-php": "^2.21",
+    "inpsyde/php-coding-standards": "^1.0.0-beta.1",
+    "php-stubs/wordpress-stubs": "^5.7",
+    "vimeo/psalm": "^4.8"
   },
   "autoload": {
-    "psr-0": {
-      "Mollie_WC_": "./src"
-    },
     "psr-4": {
+      "Mollie\\WooCommerce\\": "src/",
```

Comment:
I don't understand, do you mean change Mollie\WooCommerce\? But the plugin purpose is allowing Mollie payments for WooCommerce... Do you think it has to be more specific?

Reply:
Maybe `Mollie\\MolliePaymentsForWoocommerce`? I suggested this just because just 'WooCommerce' means literally WooCommerce, not something made for it.

But this is not something that absolutely needs to be fixed. It's just an idea to think about, nothing is broken here.